Lilu/Sources/kern_patcher.cpp

```diff
@@ -387,6 +400,14 @@ mach_vm_address_t KernelPatcher::routeFunctionInternal(mach_vm_address_t from, m
 		absolute = true;
 	}
 
+#if defined(__i386__)
+	if (jumpType != JumpType::Auto && jumpType != JumpType::Short) {
```

Comment:
Not sure this is right, as we want long jumps explicitly in many places in Lilu. Perhaps need to force short on i386, but I am really unsure.